<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkOrder</class>
 <widget class="QDialog" name="checkOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 81, 100);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_2
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_3
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_4
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_5
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_6
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_7
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 51, 60);</string>
   </property>
   <property name="text">
    <string>乘机人数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 51, 60);</string>
   </property>
   <property name="text">
    <string>总金额</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_8
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_9
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton:!hover
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:8px;
}
#pushButton:hover
{
background-color:rgb(134, 199, 255);
border-radius:8px;
color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2:!hover
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-radius:8px;
}
#pushButton_2:hover
{
background-color:rgb(134, 199, 255);
border-radius:8px;
color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-top-left-radius:15px;
border-bottom-left-radius:15px;
</string>
   </property>
   <property name="text">
    <string>订单号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_10
{
background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);
border-top-right-radius:15px;
border-bottom-right-radius:15px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>126</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 51, 60);</string>
   </property>
   <property name="text">
    <string>- &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>175</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 51, 60);</string>
   </property>
   <property name="text">
    <string>- &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>293</width>
     <height>302</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(53, 51, 60);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>lineEdit_7</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_8</zorder>
  <zorder>lineEdit_9</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_10</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
